### ISSUE TITLE
Is there a way to test this module locally on my windows without having a physical escpos printer ?

### ISSUE THREAD

**Issue by khalid-elabbadi** [song940/node-escpos#361]
**This is just a question :**
Please can any one tell me if there a way to test this module locally on my windows without having a physical escpos printer ?

**Comment by flapili** [comment]
yes you can look at rawBT android app, go to settings  => scroll to bottom and select the last button => lanShare on 9100
![image](https://user-images.githubusercontent.com/29481952/177797053-9712263e-a535-4beb-991b-b9aadd630346.png)
